easybuild/easyconfigs/v/VESTA/VESTA-3.5.8-gtk3.eb

```diff
@@ -0,0 +1,28 @@
+# Based on https://code.it4i.cz/sccs/easyconfigs-it4i/-/blob/it4i-karolina/v/VESTA/VESTA-3.5.7.eb
+
+easyblock = 'Binary'
+
+name = 'VESTA'
+version = '3.5.8'
+versionsuffix = '-gtk3'
+
+homepage = "http://www.jp-minerals.org/vesta"
+description = """
+  VESTA is a 3D visualization program for structured models, 
+  volumetric data such as electron/nuclear densities, 
+  and crystal morphologies.
+"""
+
+toolchain = SYSTEM
+
+source_urls = ["http://www.jp-minerals.org/vesta/archives/%(version)s"]
+sources = ["%(name)s-gtk3.tar.bz2"]
+checksums = ['78bef025c2b31f1d64c9c7e06ad2b139d252b3a686893ee79ac74b3021868df8']
+
+extract_sources = True
+sanity_check_paths = {
+    'files': ['VESTA'],
```

Comment:
Can we also add a sanity check command to make sure the installation works, for example (untested):

```python
sanity_check_commands = ["VESTA --help"]
```

Not sure if that's an option since this is a GUI tool...

Reply:
Any command line invocation leads to the following error:

```
$ VESTA -h
16:05:27: Error: Unable to initialize GTK+, is DISPLAY set properly?
```

Reply:
Yeah, I suspected something like that. Fine, no sanity check command then...